recipes-wpe/wpeframework/wpeframework_git.bb

```diff
@@ -9,7 +9,7 @@ require include/wpeframework.inc
 require include/compositor.inc
 
 
-DEPENDS = "zlib python-jsonref-native virtual/egl ${WPE_COMPOSITOR_DEP}"
+DEPENDS = "zlib wpeframework-tools-native virtual/egl ${WPE_COMPOSITOR_DEP}"
```

Comment:
We should also remove that python-jsonref recipe as part of this change:
https://github.com/WebPlatformForEmbedded/meta-wpe/tree/master/recipes-devtools/python-jsonref

Reply:
We still need is for the tools to work properly

Reply:
Oh right, my bad. I missed the `python-jsonref-native` in the DEPENDS above.

@kraj did my `python-jsonref` patch ever land in master?

Reply:
no i think